src/main/java/com/gluonhq/substrate/util/FileOps.java

```diff
@@ -147,4 +150,46 @@ public static Path copyFile(Path source, Path destination)  {
         }
         return destination;
     }
+
+    public static void deleteDirectory(Path start) throws IOException {
```

Comment:
add docs about what will happen if e.g. start does not exist, is null, no permission etc, and when an IOException is thrown.
There are many edge cases here, and rather than reading the code, it is easier to read the contract (e.g. JavaDoc)

Reply:
done